Packs/PhishUp/Playbooks/PhishUp_Mail_Scanner.yml

```diff
@@ -0,0 +1,743 @@
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: ea0734ac-6c94-4c7c-8e59-d14c6cbeadc9
+    type: title
+    task:
+      id: ea0734ac-6c94-4c7c-8e59-d14c6cbeadc9
+      version: -1
+      name: End Of Playbook
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 2620
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "7":
+    id: "7"
+    taskid: f6612bfc-cf2d-4f35-8268-7db53049c5f4
+    type: title
+    task:
+      id: f6612bfc-cf2d-4f35-8268-7db53049c5f4
+      version: -1
+      name: PhishUp Actions
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "15"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": -430,
+          "y": 880
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: 9774d2de-5f2b-4d2f-8b26-931d985e88cc
+    type: condition
+    task:
+      id: 9774d2de-5f2b-4d2f-8b26-931d985e88cc
+      version: -1
+      name: Detect Mail's Source Brand
+      description: Detect Mail's Source Brand
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "19"
+      Gmail:
+      - "16"
+    separatecontext: false
+    conditions:
+    - label: Gmail
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: incident.sourceBrand
+            iscontext: true
+          right:
+            value:
+              simple: Gmail
+    view: |-
+      {
+        "position": {
+          "x": -730,
+          "y": 1280
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: b2bd5841-c058-4251-8a8b-9ceb160f1fb2
+    type: regular
+    task:
+      id: b2bd5841-c058-4251-8a8b-9ceb160f1fb2
+      version: -1
+      name: Gmail Delete Mail
+      description: Deletes an email in the user's mailbox.
+      script: Gmail|||gmail-delete-mail
+      type: regular
+      iscommand: true
+      brand: Gmail
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      message-id:
+        simple: ${incident.labels.Email/ID}
+      permanent:
+        simple: "False"
+      user-id:
+        simple: ${incident.labels.Email/Header/delivered-to}
```

Comment:
Same as https://github.com/demisto/content/pull/17028/files#r785440374
Please create a playbook input and use it here.
Something like `UserID`